.github/workflows/test.yml

```diff
@@ -194,7 +194,12 @@ jobs:
       #   if: ${{ failure() && steps.unit_tests.conclusion == 'failure' }}
 
       - name: Upload coverage data
-        run: yarn coverage
```

Comment:
These tasks are not more valid as npm package `codecov` is deprecated. And they recommend to use new native binaries instead. 
https://about.codecov.io/blog/introducing-codecovs-new-uploader/